definitions/npm/express_v4.x.x/test_express_v4.x.x.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import express from 'express';
+
+const app = express();
+
+// $ExpectError property `foo` Property not found in Application:
+app.foo();
+
+const myRouter = new express.Router();
+
+myRouter.get('/dang', (req, res) => {
+    res.json({ payload: 'stuff' });
+});
+// This should work but doesn't for now... :-(
```

Comment:
Not sure why this is throwing an error. I am a bit confused on how to declare this type.